glue/app/qt/application.py

```diff
@@ -393,7 +393,8 @@ def _update_plot_dashboard(self, sub_window):
         if sub_window is None:
             return
 
-        layer_view, options_widget, title = self._get_plot_dashboards(sub_window)
+        layer_view, options_widget, title = self._get_plot_dashboards(
+            sub_window)
```

Comment:
Here's another example where I don't think wrapping makes the code more readabale